lib/core/viewmodels/quick_links_viewmodel.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_inappwebview/flutter_inappwebview.dart';
+import 'package:notredame/core/constants/quick_links.dart';
+import 'package:notredame/core/models/quick_link_model.dart';
+import 'package:notredame/ui/views/security_view.dart';
+import 'package:stacked/stacked.dart';
+
+class QuickLinksViewModel extends BaseViewModel {
+  final int numberOfLinks = quickLinks.length;
+
+  List<QuickLink> quickLinkList = quickLinks;
+
+  void onLinkClicked(BuildContext context, QuickLink links) {
+    if (links.link == 'security') {
+      Navigator.push(
+          context, MaterialPageRoute(builder: (context) => SecurityView()));
```

Comment:
Instead of doing that, you should add the call in the `router.dart` file directly with a route path like `ets/security`

Reply:
done